communication/src/PlainUDPCommunication.cpp

```diff
@@ -71,6 +71,9 @@ class PlainUDPCommunication::PlainUdpImpl {
 
   NodeNum selfId;
 
+  // Max UDP packet bytes can be sent.
+  const uint16_t dataLimit = 65507;
```

Comment:
Where did this number come from? 2^16 == 65536.

Reply:
UDP datagram composed of:
65507 data bytes
20 bytes IP header
8 bytes UDP header

Reply:
Let's make this more explicit (and static), as in `static constexpr uint16_t MAX_UDP_PAYLOAD_SIZE = 65535 - 20 - 8`